index.html

```diff
@@ -0,0 +1,1305 @@
+              </h5>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="col-lg-4 col-md-4 col-sm-12 col-xm-12 w-100">
+        <div class="media more-stories pb-3">
+          <img src="images/deck_img.jpg" class="mapping-small lazysize lazyloaded mr-3" alt="newsweek-img">
+
+          <div class="media-body">
+            <h5 class="media-title">More Aggressive' Coronavirus Strain May Have Evolved—But Don't Worry</h5>
+
+          </div>
+        </div>
+        <div class="media mt-3 more-stories pb-3">
+          <img src="images/deck_img.jpg" class="mapping-small lazysize lazyloaded mr-3" alt="newsweek-img">
+
+          <div class="media-body">
+            <h5 class="media-title">More Aggressive' Coronavirus Strain May Have Evolved—But Don't Worry</h5>
+
+          </div>
+        </div>
+        <div class="media mt-3 more-stories side pb-3">
+          <img src="images/deck_img.jpg" class="mapping-small lazysize lazyloaded mr-3" alt="newsweek-img">
+
+          <div class="media-body">
+            <h5 class="media-title">More Aggressive' Coronavirus Strain May Have Evolved—But Don't Worry</h5>
+
+          </div>
+        </div>
+      </div>
+    </div>
+  </main>
+
+  <div class="container-fluid mt-1">
+    <div class=" row pl-lg-4 pr-lg-4 bg-white ">
+      <section class=" col-lg-8 mt-5 col-md-12">
+        <h5>SUBSCRIBE</h5>
+        <div class="jumbotron d-flex">
+          <div class="d-flex">
+            <img
+              src="https://d.newsweek.com/en/full/1569344/march-06-2020-cover.jpg?w=170&amp;q=95&amp;f=55944e2b38c7b505782d06582edd44fd"
+              class="img-fluid mb-3" alt="newsweek-img">
+
+            <img
+              src="https://d.newsweek.com/en/full/1572460/march-13-2020.jpg?w=170&amp;q=95&amp;f=52ec8e410e9337459df2777847b09b11"
+              class="img-fluid jumbo-img mt-4 " alt="newsweek-img">
+          </div>
+          <div class="ml-5 ">
+            <h4 class="ml-5 mt-4">In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h4>
+            <p class="ml-5">All Access + Weekly Delivery.</p>
+            <p class="ml-5">
+              Choose A Membership That's Perfect for You!
+            </p>
+            <button class="ml-5 footer-button form-control text-white font-weight-bold">SUBSCRIBE</button>
+          </div>
+        </div>
+      </section>
+      <section class=" col-lg-4 mt-5 col-md-12">
+        <h5>NEWSLETTER</h5>
+        <div class="card mb-3 sides">
+          <div class="row no-gutters">
+            <div class="col-md-6">
+              <div class="start-img img-fluid w-100"></div>
+            </div>
+            <div class="col-md-6">
+              <div class="card-body">
+                <h4 class="card-title">The Starting 5</h4>
+                <p class=" card-text start-text">Take back your inbox. Sign up for our newsletter and start making sense
```

Comment:
The newsletter section looks taller in the medium and the small breakpoints. Please fix this.
yours:
![Screenshot from 2020-03-14 18-39-19](https://user-images.githubusercontent.com/51926937/76692924-3fe3be80-6623-11ea-81fd-adfd2d19bbd7.png)

original:
![Screenshot from 2020-03-14 18-39-14](https://user-images.githubusercontent.com/51926937/76692927-43774580-6623-11ea-86d1-4f01bab62e61.png)

Reply:
fixed, by making it smaller.